src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/Button/ButtonHandler.iOS.cs

```diff
@@ -9,7 +8,8 @@ public partial class ButtonHandler : AbstractViewHandler<IButton, UIButton>
 
 		public static void MapText(IViewHandler handler, IButton view)
 		{
-			((UIButton)handler.NativeView).SetText(view.Text);
+			if (handler.NativeView is UIButton uIButton)
+				uIButton.SetText(view.Text);
```

Comment:
checking that NativeView isn't null isn't enough, we probably should check wether it's disposed. Or are setting NativeView to null when we dispose it ?

also, why do we need casting ? don't we have a generic type parameter of type UIButton ?